### ISSUE TITLE
section annotation is going the wrong way. 

### ISSUE THREAD

**Issue by theoryshaw** [IfcOpenShell/IfcOpenShell#4103]
In the following file the section annotation (marked in red) is going the wrong way. 

file: https://hub.openingdesign.com/OpeningDesign/Restaurant_Sun_Prairie/src/commit/7a8f0840457a6923a92a41fdb98cf54f3a988942/Models/BlenderBIM/Restaurant_Sun_Prairie.ifc

![image](https://github.com/IfcOpenShell/IfcOpenShell/assets/507113/e1bd9b62-63fb-407e-8da0-888fd7326183)
![2023-12-11_20-16-04_blenderBlender_ DDropboxGitea_ODRestaurant_Sun_PrairieModblender](https://github.com/IfcOpenShell/IfcOpenShell/assets/507113/cded4ed5-5e72-4bc3-967c-e92a17acc5a0)

**Comment by theoryshaw** [comment]
This might have been due to me duplicating a section view, and rotating it 180 degrees to use as another section.

**Comment by Georift** [comment]
I'm actively running into this problem now. Did you ever run into this again? I don't recall ever rotating the section views. Deleting and recreating it still shows an incorrect direction.

**Comment by theoryshaw** [comment]
Yes, this still happens to me, from time to time.  I haven't figured out what causes it yet. 

I use this little plugin to flip the vertices: https://community.osarch.org/discussion/comment/25750/#Comment_25750
